.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
   "cSpell.words": ["bytecode", "CNCF", "Emscripten", "wasmedge"]
-}
+}
```

Comment:
Can we revert the change to this file?

Reply:
I tried but its not working. :3

Reply:
Sus change, lol

Reply:
Please `git rebase` your commit to discard the changes of this file.
https://stackoverflow.com/questions/1186535/how-do-i-modify-a-specific-commit